pkg/virt-launcher/virtwrap/converter/converter_test.go

```diff
@@ -233,7 +238,7 @@ var _ = Describe("Converter", func() {
 
 		It("should not set disk I/O mode if not requested", func() {
 			v1Disk := &v1.Disk{}
-			xml := diskToDiskXML(v1Disk)
+			xml := diskToDiskXML(runtime.GOARCH, v1Disk)
```

Comment:
I think we shouldn't use `runtime.GOARCH` here for these reasons:
1. Is the expectedXML in this test expected for all the architectures?
2. The call to `diskToDiskXML` will fail for an unsupported arch. Theoretically, someone working on an unsupported arch could be developing Kubevirt and still expect to run unit tests successfully.
3. In general, it's unhealthy to use actual non-mocked attributes that depend on the running system during unit tests.

I think it would be best to turn this into a describe table as you did for the other tests. WDYT?

Reply:
Done!

Reply:
Just to answer here:
> Is the expectedXML in this test expected for all the architectures?

yes, the output is the same here